CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.0.3
```

Comment:
In general, I don't think it is necessary to bump the version for each commit. Is there a reason we are doing this?

Reply:
In this particular case, I want to upgrade flutter to use right link.
In general, if it is just documentation, I am skipping it.
Added test coverage for links to this PR.